servers/audio/effects/audio_effect_record.cpp

```diff
@@ -250,8 +255,12 @@ Ref<AudioStreamSample> AudioEffectRecord::get_recording() const {
 		Vector<uint8_t> bleft;
 		Vector<uint8_t> bright;
 
+#ifdef TOOLS_ENABLED
 		ResourceImporterWAV::_compress_ima_adpcm(left, bleft);
 		ResourceImporterWAV::_compress_ima_adpcm(right, bright);
+#else
+		ERR_PRINT("AudioEffectRecord cannot do IMA ADPCM compression at runtime.");
+#endif
```

Comment:
This will have to be reworked @godotengine/audio.

Either `ResourceImporterWAV::_compress_ima_adpcm` should be moved to `core` or `servers`, or this should be changed to ensure this is editor only code (and thus moved to an import plugin?).